photo-app/app/models/payment.rb

```diff
@@ -0,0 +1,20 @@
+class Payment < ApplicationRecord
+  attr_accessor :card_number, :card_cvv, :card_expires_month, :card_expires_year
+  belongs_to :user
+
+  def self.month_options
+    Date::MONTHNAMES.compact.each_with_index.map { |name, i| ["#{i+1} - #{name}", i+1]}
```

Comment:
Don't use single character as variable names.
Why?

Reply:
Hard to know what it is if the code base becomes larger?

Reply:
that is one. secondly, if you want to edit that variable, you can't easily select it. thirdly, with a proper name for the variable, everyone can easily understand what that variable means

Reply:
i have changed the variable name from "name" to "month" and "i" to "index"